roles/ohpc_install/templates/slurm_conf.j2

```diff
@@ -108,3 +108,8 @@ NodeName=c0 Sockets=1 CoresPerSocket=1 State=UNKNOWN
 #PartitionName=high Nodes=compute-[0-1] Default=YES MaxTime=INFINITE State=UP PriorityTier=10
 #PartitionName=gpu  Nodes=gpu-compute-1 Default=YES MaxTime=INFINITE State=UP PriorityTier=5 AllowGroups=slurmusers
 PartitionName=low  Nodes=c0 Default=YES MaxTime=2-00:00:00 State=UP
+
+#OOD
+{% if ood_provision %}
+	NodeName=={{ ood }}
```

Comment:
Check and make sure this leading white space on the NodeName is correct syntax.   I suggest removing it regardless.  Keep in mind that we aren't doing 'good coding block indent patterns' with these templates.  They are literally just inserting text into a config file.  Unless the white space makes sense in the resulting config file, it likely shouldn't be in the template.